xapian-letor/ranker/ranker.cc

```diff
@@ -192,32 +192,29 @@ write_to_file(const std::vector<Xapian::FeatureVector> & list_fvecs, const strin
     }
 }
 
+// Query file is in the format: <qid> '<query_string>'
```

Comment:
This isn't quite true. As the parser is written, it will accept arbitrary characters between the first space (`j` up to line 207) and the first single quote (`i` from line 206). This is fine (it can cope with arbitrary numbers of spaces between the two terms) but perhaps needs a little note in the comment.

Reply:
fixed in a fixup commit.